specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/JavaComponents.json

```diff
@@ -0,0 +1,481 @@
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "JavaComponents"
+        ],
+        "summary": "Update properties of a Java Component",
+        "description": "Patches a Java Component using JSON Merge Patch",
+        "operationId": "JavaComponents_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Java Component.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "javaComponentEnvelope",
+            "in": "body",
+            "description": "Configuration details of the Java Component.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/JavaComponent"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/JavaComponent"
+            }
+          },
+          "202": {
+            "description": "Patch operation is in progress.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patch Java Component": {
+            "$ref": "./examples/JavaComponents_Patch.json"
+          },
+          "Patch Java Component with ServiceBinds": {
+            "$ref": "./examples/JavaComponents_Patch_ServiceBind.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "JavaComponents"
+        ],
+        "summary": "Delete a Java Component.",
+        "description": "Delete a Java Component.",
+        "operationId": "JavaComponents_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
please always use the latest, current is "v5". please also update other places

Reply:
Hi @ms-henglu I have changed them to v5